test/unit/iframe.js

```diff
@@ -1,94 +1,271 @@
+    });
+
+    it('disables scrolling, allows transparency, and removes border in iframe\'s attributes', () => {
+      iframe = new Iframe(parent, constructorConfig);
+      assert.equal(iframe.el.getAttribute('horizontalscrolling'), 'no');
+      assert.equal(iframe.el.getAttribute('verticalscrolling'), 'no');
+      assert.equal(iframe.el.getAttribute('allowTransparency'), 'true');
+      assert.equal(iframe.el.getAttribute('frameBorder'), '0');
+      assert.equal(iframe.el.getAttribute('scrolling'), 'no');
     });
 
-    it('appends an iframe', () => {
-      assert.equal('IFRAME', parent.children[0].tagName);
+    it('sets element name to name in config', () => {
+      assert.equal(iframe.el.getAttribute('name'), constructorConfig.name);
     });
 
-    it('appends style tag to head', () => {
-      const styleTags = iframe.document.head.getElementsByTagName('style');
-      assert.equal(styleTags.length, 1);
+    it('sets styleTag to null', () => {
+      iframe = new Iframe(parent, constructorConfig);
+      assert.equal(iframe.styleTag = null);
     });
   });
 
-  describe('loadFontScript', () => {
-    let scriptTags;
+  describe('prototype methods', () => {
+    let parent;
+    let iframe;
+
     beforeEach(() => {
-      scriptTags = document.head.getElementsByTagName('script').length;
-      iframe.googleFonts = ['Lato'];
-      return iframe.load();
+      parent = document.createElement('div');
+      parent.setAttribute('id', 'fixture');
+      document.body.appendChild(parent);
+      iframe = new Iframe(parent, configObject);
     });
 
-    it('appends a script tag', () => {
-      const newScriptTags = document.head.getElementsByTagName('script');
-      assert.equal(newScriptTags.length, scriptTags + 1);
+    afterEach(() => {
+      document.body.removeChild(parent);
     });
-  });
 
-  describe('get css', () => {
-    it('returns properly formatted CSS', () => {
-      return iframe.load().then(() => {
-        assert.include(iframe.css, defaultCSS, 'css is formatted correctly');
-        assert.include(iframe.css, customCSS, 'appends custom css');
+    describe('load()', () => {
+      let loadFontsStub;
+      let appendStyleTagStub;
+
+      beforeEach(() => {
+        loadFontsStub = sinon.stub(iframe, 'loadFonts').resolves();
+        appendStyleTagStub = sinon.stub(iframe, 'appendStyleTag').resolves();
+      });
+
+      afterEach(() => {
+        loadFontsStub.restore();
+        appendStyleTagStub.restore();
+      });
+
+      it('appends a single iframe to parent', async () => {
+        await iframe.load();
+        assert.equal(parent.children.length, 1);
+        assert.equal(parent.children[0].tagName, 'IFRAME');
+      });
+
+      it('loads fonts then append style tag on iframe load', async () => {
+        await iframe.load();
+        assert.calledOnce(loadFontsStub);
+        assert.calledOnce(appendStyleTagStub);
+        await iframe.el.onload();
```

Comment:
Since `appendChild()` will trigger the element's `onload()`, I think we can omit the explicit `iframe.el.onload()`.